airflow/www/views.py

```diff
@@ -1855,6 +1855,19 @@ def trigger(self, session=None):
             flash(f"Cannot create dagruns because the dag {dag_id} has import errors", "error")
             return redirect(origin)
 
+        recent_confs = (
+            session.query(DagRun.conf)
+            .filter(
+                DagRun.dag_id == dag_id,
+                DagRun.run_type == DagRunType.MANUAL,
+                DagRun.conf is not None,
```

Comment:
I don't think empty objects are relevent here. (also update the `is not None` syntax)
```suggestion
                DagRun.conf.isnot(None),
                DagRun.conf != {}
```